File: auth-center/src/main/java/com/jmsoftware/maf/authcenter/user/mapper/UserMapper.java
package com.jmsoftware.maf.authcenter.user.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import com.jmsoftware.maf.authcenter.user.entity.persistence.User;
import org.apache.ibatis.annotations.Mapper;

/**
 * <h1>UserMapper</h1>
 * <p>
 * Mapper of UserPersistence.(UserPersistence)
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com
 * @date 5 /10/20 12:17 PM
 */
@Mapper
public interface UserMapper extends BaseMapper<User> {
}

File: auth-center/auth-center-infra/src/main/kotlin/com/jmsoftware/maf/authcenter/user/mapper/UserMapper.kt
package com.jmsoftware.maf.authcenter.user.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import com.jmsoftware.maf.authcenter.user.persistence.User
import org.apache.ibatis.annotations.Mapper

/**
 * # UserMapper
 *
 * Mapper of UserPersistence.(UserPersistence)
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/12/22 12:45 PM
 */
@Mapper
interface UserMapper : BaseMapper<User>
